Makefile

```diff
@@ -3,6 +3,13 @@ CXX = c++
 CXXFLAGS = -Wall -Wextra -Werror -std=c++98 -I. -Iinclude -Isrc -g
 DEPFLAGS = -MMD -MP
 
+UNAME_S := $(shell uname -s)
```

Comment:
That's for your mac, right ?

Can you explain why are you using fsanitize and is it slowing the webserver significantly ?

Reply:
I had a segfault (only on Mac), cause of the different way to manage memory among the two systems, this requestTarget thing you commented on. We don't have Valgring on Mac but this fsanitize. We can comment it out, we don't need it actually.

Reply:
We can change the Makefile so that that flag is loaded only if make is started with a special keyword or just comment it out. And we will comment it in, if we need it. Let me know what you think. @lmangall

Reply:
I see, a keyword would be ideal but commenting in/out is enough.

Did you find fsanitize helpful ? When I was looking for a good Valgrind equivalent, a long time ago, I didn't liked any of the tools. Leaks was a lot of things to add to the code to make it work, fsanitize didn't had precise information...

Reply:
Yes, fsanitize was helpful. I managed to fix the leak.

Reply:
Thanks, good to know